addons/account/views/account_move_views.xml

```diff
@@ -914,7 +913,7 @@
                                        invisible="move_type not in ('out_invoice', 'out_refund', 'out_receipt')"/>
                                 <label for="partner_id" string="Vendor" style="font-weight:bold;"
                                        invisible="move_type not in ('in_invoice', 'in_refund', 'in_receipt')"/>
-                                <div class="o_col">
+                                <div class="o_col" invisible="move_type not in ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt')">
```

Comment:
we could use move_type != entry but i copy paste the invisible condition of the partner_id field to be consistent

Reply:
has been r+ already in another pr

Reply:
Isn't it a useless diff then ?

Reply:
rebase issues oopsss